pretext/cli.py

```diff
@@ -419,12 +409,6 @@ def build(
         log.debug(e, exc_info=True)
         return
 
-    # Call generate if flag is set
-    if generate:
```

Comment:
This was running even when `no_generate == True`. But we're not really respecting what asset was requested on the command line anyway. I think if you want to manually generate assets, just run `pretext generate`.